src/Symfony/Component/Workflow/Validator/DefinitionValidator.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+namespace Symfony\Component\Workflow\Exception;
+
+use Symfony\Component\Workflow\Definition;
+
+/**
+ * @author Tobias Nyholm <tobias.nyholm@gmail.com>
+ */
+interface DefinitionValidator
```

Comment:
In Symfony, we suffix interfaces with `Interface` => `DefinitionValidatorInterface`